client/src/components/JurisdictionAdmin/RoundManagement.test.tsx

```diff
@@ -68,6 +89,15 @@ const apiCalls = {
 }
 
 describe('RoundManagement', () => {
+  beforeEach(() => {
+    // Clear mock call counts, etc.
+    jest.clearAllMocks()
+
+    // Reset the BlobProvider mock's implementation since some tests override the default mock
+    // implementation
+    ;(BlobProvider as jest.Mock).mockImplementation(() => null)
```

Comment:
This funky leading semicolon is a side effect of the fact that Arlo's auto-formatting prefers no trailing semicolons. The leading semicolon here is necessary to distinguish from a function call (`jest.clearAllMocks()(BlobProvider as jest.Mock)...`)